### DISCUSSION TITLE
TimeMaster.inverseFixedTimeFactor necessary?

### DISCUSSION THREAD

**Discussion by Laur3nt1u** [JustInvoke/Randomation-Vehicle-Physics#52] [General]
Isn't TimeMaster.inverseFixedTimeFactor just the Time.fixedUnscaledDeltaTime * 100?

**Comment by JustInvoke** [comment]
I think so. I remember writing the TimeMaster script to help with adjusting the fixed timestep based on the timescale because slowing the timescale would cause a noticeable framerate drop in terms of fixed updates since they would be further spaced out. It's also used for variable fixed update rates to allow the physics to act consistently with different values. I don't know if fixedUnscaledDeltaTime was available when I wrote it but it might be possible to use it instead.
